packages/dataviz/package.json

```diff
@@ -35,27 +35,34 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "d3": "^6.3.0",
+    "d3": "^6.5.0",
     "date-fns": "^2.16.1",
-    "recharts": "^1.8.5"
+    "geojson": "^0.5.0",
+    "lodash": "^4.17.20",
+    "recharts": "^1.8.5",
```

Comment:
do you think we can upgrade to recharts 2 ?
recharts 1 is based on a part of d3 5 so you have both d3 version today

Reply:
Thanks for sharing this, no idea yet, we need to give a try and to evaluate the impact. This should be done in another dedicated PR to separate topics and to allow to move forward on smaller topics.